docs/docs/metrics.md

```diff
@@ -138,6 +138,9 @@ Dashes in metric names are replaced with underscores.
   received since the current Marathon instance became a leader.
 * `marathon.mesos.offers.used.counter` — the count of offers used since
   the current Marathon instance became a leader.
+* `marathon.mesos.task-states.<task-state>.counter` — the count of task
```

Comment:
I'm more in favor of putting "updates" in the metric. `marathon.mesos.task-updates.<task-state>.counter`